### DISCUSSION TITLE
Verschiedene Verkehrsverbünde zum auswählen hinzufügen

### DISCUSSION THREAD

**Discussion by Niklas1012** [Traewelling/traewelling#2782] [Ideas / Feature Requests]
Hallo,

ich hätte einen Vorschlag bezüglich der Datenquelle von Träwelling.

Ich selber nutze Träwelling sehr oft und gerne, danke dafür.

Da aber Träwelling den DB Navigator/ Api als Datenquelle verwendet, werden dementsprechend nicht alle Verkehrsunternehmen richtig in den Statistiken und generell angezeigt.
Auch die Echtzeitdaten werden dadurch nicht immer richtig berechnet.

Mein Vorschlag wäre es daher, eine Rubrik in den Einstellungen hinzufügen, bei der man den jeweiligen Verkehrsverbund (RMV/ NVV/ VBB...) auswählen kann damit die Daten richtig angezeigt werden können. 

Mit freundlichen Grüßen
Niklas Müller

**Comment by pascalwittler** [comment]
Kannst du ein Beispiel (z.B. mit Screenshot) geben, wo ein Verkehrsverbund nicht so dargestellt wird, wie du es gerne hättest und was stattdessen dein Wunschzustand wäre?

Aus deinem Text bin ich mir nicht sicher, ob du eine andere Datenquelle als den DB Navigator / die HAFAS-API verwenden möchtest oder ob du "nur" die grafische bzw. textuelle Darstellung an irgendeiner Stelle ändern (in deinen Benutzereinstellungen überschreiben?) möchtest.

**Comment by Niklas1012** [comment]
Viele Verkehrsunternehmen werden nicht im RMV Gebiet von Träwelling mit den Namen erfasst und erscheinen somit nicht in den Statistiken was schade ist.
Diese werden alle zusammen als "Sonstige" zusammengeführt.

Auch für höhere Genauigkeit der Daten wäre es Sinnvoll das man als Feature in den Einstellungen verschiedene Verkehrsverbünde wie in diesem Fall den RMV auswählen könnte.
Sprich den Verkehrsverbund auswählen in dem man regelmäßig fährt.
![Screenshot_20240722_182440_Chrome](https://github.com/user-attachments/assets/9385f652-f5ae-46ca-8a7e-b084e14e967e)
